bin/.any-nix-wrapper

```diff
@@ -5,12 +5,18 @@ fns () {
     which_shell="$1"
     shift
     pos=0
+
+    subcommand=shell
+    if nix --version | grep -q ' 2\.[0-3][^0-9]'; then
```

Comment:
What was your motivation for using `[^0-9]` instead of `\.`? (Just asking. Maybe there's a good reason.)

Reply:
The small possibility that somebody might be running 2.3preWHATEER